inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/db/JobProfileDb.java

```diff
@@ -192,4 +197,20 @@ public List<JobProfile> getJobsByState(StateSearchKey stateSearchKey) {
         }
         return profileList;
     }
+
+    /**
+     * check local job state.
+     *
+     * @return list of job profile.
```

Comment:
The result type is a Map, please change the Java doc, and add the desc for the key-value.

Reply:
done